cif/products/src/mappers/PagedResponseProductMapper.ts

```diff
@@ -112,4 +118,25 @@ export default class PagedResponseProductMapper extends Mapper<PagedResponseProd
       totalResults,
     });
   }
+
+  private buildCIFRequest(mappable: any) {
```

Comment:
we are inside a mapper, maybe this method can be called mapFilter and simply return the mapped filter instead of returning the complete object, since you are duplicating code.

Reply:
renamed to mapFilter and now only returns the mapped filter for query